### ISSUE TITLE
addWMS() does not hide the attribution when corresponding group is hidden

### ISSUE THREAD

**Issue by florisvdh** [trafficonese/leaflet.extras2#55]
After adding a WMS layer with` addWMS()`, the attribution display at the bottom is not toggled when the group is shown or hidden respectively. This contrasts with such toggle-behaviour in `leaflet::addWMSTiles()`, where the attribution is hidden if the group is hidden.

The attached zipped html notebook – [leaflet_issue_notebook.nb.html.zip](https://github.com/trafficonese/leaflet.extras2/files/10838871/leaflet_issue_notebook.nb.html.zip) – compares this behaviour between **leaflet** (first map) and **leaflet.extras2** (second map).

I believe **leaflet.extras2** should follow the behaviour of **leaflet** for this.

**Comment by trafficonese** [comment]
You're right, but I do think the bug comes from the underlying plugin library. 
I will have a deeper look into it, or you could open an Issue here https://github.com/heigeo/leaflet.wms

**Comment by trafficonese** [comment]
Attribution is on the WMS layer now and hides with the group, same as `addWMSTiles()`.
Example: `inst/examples/wms_app.R`
